src/Foundation/NSAction.cs

```diff
@@ -55,35 +58,42 @@ public void Apply ()
 	}
 
 	// Use this for synchronous operations
-	[Register ("__MonoMac_ActionDispatcher")]
-	internal sealed class ActionDispatcher : NSObject {
+	[Register ("__MonoMac_NSActionDispatcher2")]
+	internal sealed class NSActionDispatcher2 : NSObject
```

Comment:
Why was the class renamed? It doesn't look necessary, and it makes the diff harder to understand.

Reply:
This wasn't renamed. Check commit-by-commit. It removed an unused class.